pkg/tool/gerrit/client_test.go

```diff
@@ -0,0 +1,26 @@
+package gerrit
```

Comment:
Add license for golang files. You can refer to https://github.com/koderover/zadig/blob/main/pkg/microservice/aslan/core/code/client/gerrit/gerrit.go